### PR TITLE
IO - Fix non-ASCII characters breaking channel switching

### PR DESCRIPTION
**Pull request by jokoho48**
**When merged this pull request will:**
- Add Sanitization of serverName in sqf to prevent ts3ChannelMove to fail

### CODE REVIEW

**Comment by jonpas** [comment]
Does this fix #579?

**Comment by jokoho48** [comment]
> Does this fix #579?

Yes, it should fix that issue.

File: addons/sys_io/fnc_ts3ChannelMove.sqf [hunk 1]
```diff
@@ -15,6 +15,10 @@
  * Public: No
  */
 
-private _ts3ChannelDetails = format ["%1,%2,%3", EGVAR(sys_core,ts3ChannelName), EGVAR(sys_core,ts3ChannelPassword), serverName];
+private _serverName = toString ((toArray serverName) select {_x < 127});
+if (_serverName == "") then {
+    _serverName = "ServerName Fully Sanitized away";
```
Comment (jonpas) [inline addons/sys_io/fnc_ts3ChannelMove.sqf]: Can we add a warning for RPT for that as well?

Comment (jokoho48) [reply addons/sys_io/fnc_ts3ChannelMove.sqf]: done

File: addons/sys_io/fnc_ts3ChannelMove.sqf [hunk 2]
```diff
@@ -15,6 +15,11 @@
...
+    WARNING_1("Server name '%1' did not include any ASCII Characters and got fully Sanaitzed.", serverName)
```
Comment (jonpas) [inline addons/sys_io/fnc_ts3ChannelMove.sqf]:
```suggestion
    _serverName = "Unsupported Server Name";
    WARNING_1("Server name '%1' did not include any valid (ASCII) characters and got fully sanitized!",serverName)
```

**Review by jonpas — APPROVED** [review]

**Pull request merged**